libs/ui/src/print_element.test.tsx

```diff
@@ -29,110 +30,110 @@ const fakeOptions: PrintOptions = { sides: 'one-sided' };
 
 describe('printElement', () => {
   test('calls print with expected args', async () => {
-    await printElement(simpleElement, fakeOptions);
-    expect(printer.print).toHaveBeenCalledTimes(1);
-    expect(printer.print).toHaveBeenLastCalledWith(
-      expect.objectContaining(fakeOptions)
-    );
-  });
-
-  test('renders element within print container', async () => {
-    const printPromise = printElement(simpleElement, {
-      sides: 'one-sided',
-    });
-
-    await waitFor(async () => {
-      const printContainer = await screen.findByTestId('print-root');
-      within(printContainer).getByText('Print me!');
+    await suppressingConsoleOutput(async () => {
+      await printElement(simpleElement, fakeOptions);
+      expect(printer.print).toHaveBeenCalledTimes(1);
+      expect(printer.print).toHaveBeenLastCalledWith(
+        expect.objectContaining(fakeOptions)
+      );
     });
```

Comment:
RTL really doesn't like how we're testing this `printElement` component so I had to wrap all tests in this file with `suppressingConsoleOutput`. I don't like it and I tried to find lots of other ways to do this, but the `act` warnings are brutal and if wrap in asynchronous `act` calls the tests just don't complete.

Reply:
Happy to investigate this more with you or just leave as is

Reply:
I spent a while with this one and originally wrote the tests. I think it's best to just leave as-is, but feel free if you're interested.